packages/blocks/src/page-block/default/selection-manager.ts

```diff
@@ -385,6 +385,12 @@ export class PageSelectionState {
     }
   }
 
+  blur() {
+    resetNativeSelection(null);
+    // deactivate quill keyboard event handler
+    (document.activeElement as HTMLElement).blur();
```

Comment:
> The [Element](https://developer.mozilla.org/en-US/docs/Web/API/Element) which currently has focus, [<body>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) or null if there is no focused element.
>
> https://developer.mozilla.org/en-US/docs/Web/API/Document/activeElement


This cast is unsafe, you should check it first
```suggestion
    if (
      document.activeElement &&
      document.activeElement instanceof HTMLElement
    ) {
      document.activeElement.blur();
    }
```